project-metrics/metrics_service/database/models.py

```diff
@@ -82,7 +82,7 @@ class Build(Base):
 
   @classmethod
   def is_last_90_days(cls):
```

Comment:
I gotta admit when I first saw this my mind drifted to "I don't like that there's a hard coded 90 in this method name", but that doing a `method_missing` thing here with dynamic number parsing would be so, so much worse despite how cool it would be 😅
https://dev.to/rpalo/metaprogramming-python---method-missing

Reply:
Yeah I wavered a bit on that too, but since pretty much all of the metrics have the 90 day limit as an integral component of their definition, it seemed reasonable.